modular_skyrat/master_files/code/modules/clothing/anthro_clothes.dm

```diff
@@ -2,7 +2,7 @@
 
 //HEAD>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
god I hate these things so much, it's like the worst way to section things because I always get hit by that when looking for merge conflicts.

Reply:
It is the way, simply search for <<<<